plugins/mcreator-localization/lang/texts.properties

```diff
@@ -3571,4 +3573,4 @@ debug.threads.status=Group: {0}, status: {1}
 debug.markers.client=Client hits: {0}
 debug.markers.server=Server hits: {0}
 debug.markers.other=Other hits: {0}
-debug.frames.no_frames=Stack frames load on breakpoint hit
\ No newline at end of file
+debug.frames.no_frames=Stack frames load on breakpoint hit
```

Comment:
Please remove this newline change

Reply:
how did that even generate?!

but ofcourse, ill remove it

Reply:
i removed it!

Reply:
It can happen when using github online editor in some cases